invoice/src/main/java/org/killbill/billing/invoice/usage/ContiguousIntervalUsageInArrear.java

```diff
@@ -184,21 +188,24 @@ public UsageInArrearItemsAndNextNotificationDate computeMissingItemsAndNextNotif
         final List<RolledUpUsage> allUsage = getRolledUpUsage();
         for (final RolledUpUsage ru : allUsage) {
 
+            List<ToBeBilledConsumableInArrearDetail> toBeBilledUsageDetails = Lists.newLinkedList();
             BigDecimal toBeBilledUsage = BigDecimal.ZERO;
             if (usage.getUsageType() == UsageType.CAPACITY) {
                 toBeBilledUsage = computeToBeBilledCapacityInArrear(ru.getRolledUpUnits());
```

Comment:
Should we also add the details for `UsageType.CAPACITY`. This is simpler since there will be only one item, but writing the detail about the chosen tier might be useful.